File: core/core-api/src/main/java/com/coffee/gu/api/controller/v1/PaymentViewController.java
package com.coffee.gu.api.controller.v1;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
public class PaymentViewController {

    @GetMapping("/payment-test")
    public String paymentTest() {
        return "forward:/payment-test.html";
    }

    @GetMapping("/payment-success")
    public String paymentSuccess() {
        return "forward:/payment-success.html";
    }

    @GetMapping("/payment-fail")
    public String paymentFail() {
        return "forward:/payment-fail.html";
    }
}

File: core/core-api/src/main/kotlin/com/coffee/gu/api/controller/v1/PaymentViewController.kt
package com.coffee.gu.api.controller.v1

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping

@Controller
class PaymentViewController {
    @GetMapping("/payment-test")
    fun paymentTest(): String {
        return "forward:/payment-test.html"
    }

    @GetMapping("/payment-success")
    fun paymentSuccess(): String {
        return "forward:/payment-success.html"
    }

    @GetMapping("/payment-fail")
    fun paymentFail(): String {
        return "forward:/payment-fail.html"
    }
}
